### PR TITLE
Specify Dockerfile section in Dev Environments

### PR DESCRIPTION
**Pull request by felipecruz91**
### Proposed changes

Specify Dockerfile section in Dev Environments

### CODE REVIEW

File: desktop/dev-environments.md [hunk 1]
```diff
@@ -243,9 +243,19 @@ The `development`target uses a `golang:1.16-alpine` image with all dependencies
 
 In our example, the Docker Compose files are the same. However, they could be different and the services defined in the main Compose file may use other targets to build or directly reference other images.
 
+## Specify a Dockerfile 
+
+In this preview, Dev Environments support a simple YAML file which allows you to specify a Dockerfile to define your Dev Environment. You must include this as part of the `.docker` folder and then add it as a `config.json` file. For example:
```
Comment (glours) [inline desktop/dev-environments.md:248]:
```suggestion
In this preview, Dev Environments support a simple JSON file which allows you to specify a Dockerfile to define your Dev Environment. You must include this as part of the `.docker` folder and then add it as a `config.json` file. For example:
```

**Review by glours — COMMENTED** [review]
How did we miss that one 🤦‍♂️

File: desktop/dev-environments.md (hunk 1 above)
Comment (rumpl) [reply desktop/dev-environments.md:248]: Nothing is simple 
```suggestion
In this preview, Dev Environments support a YAML file which allows you to specify a Dockerfile to define your Dev Environment. You must include this as part of the `.docker` folder and then add it as a `config.json` file. For example:
```

File: desktop/dev-environments.md [hunk 2]
```diff
@@ -243,9 +243,19 @@ The `development`target uses a `golang:1.16-alpine` image with all dependencies
...
+
+```jsx
```
Comment (thaJeztah) [inline desktop/dev-environments.md:250]: s/jsx/json/ ? (not sure if rouge  understands jsx)

**Review by glours — APPROVED** [review]

File: desktop/dev-environments.md [hunk 3]
```diff
@@ -243,9 +243,19 @@ The `development`target uses a `golang:1.16-alpine` image with all dependencies
...
+{
+    "dockerfile": "Dockerfile.devenv"
+}
+```
+
 ## Specify a base image
 
-In this preview, Dev Environments support a simple YAML file which allows you to specify the base image that you would like to use as part of your Dev Environment. You must include this as part of the `.docker` folder and then add it as a `config.json` file. For example, to use the Jekyll base image, add:
+In the case that you already have a image built, you can specify it as a base image to define your Dev Environment. You must include this as part of the `.docker` folder and then add it as a `config.json` file. For example, to use the Jekyll base image, add:
```
Comment (gtardif) [inline desktop/dev-environments.md:258]:
```suggestion
If you already have an image built, you can specify it as a base image to define your Dev Environment. You must include this as part of the `.docker` folder and then add it as a `config.json` file. For example, to use the Jekyll base image, add:
```

File: desktop/dev-environments.md [hunk 4]
```diff
@@ -243,9 +243,19 @@ The `development`target uses a `golang:1.16-alpine` image with all dependencies
 
 In our example, the Docker Compose files are the same. However, they could be different and the services defined in the main Compose file may use other targets to build or directly reference other images.
 
+## Specify a Dockerfile 
+
+In this preview, Dev Environments support a simple JSON file which allows you to specify a Dockerfile to define your Dev Environment. You must include this as part of the `.docker` folder and then add it as a `config.json` file. For example:
```
[the remark this answers is not in the document]
Comment (felipecruz91) [reply desktop/dev-environments.md:248]:
```suggestion
In this preview, Dev Environments support a JSON file which allows you to specify a Dockerfile to define your Dev Environment. You must include this as part of the `.docker` folder and then add it as a `config.json` file. For example:
```

File: desktop/dev-environments.md (hunk 2 above)
Comment (rumpl) [reply desktop/dev-environments.md:250]: Looks like we already used `jsx` in the [base image example](https://github.com/docker/docker.github.io/pull/13973/files#diff-1c418e0ee39feb65f4c22b9a4dca3b1d4723f00e3140a26cf909c0effebc3815R260)

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**